File: 250312/숫자로 이루어진 사각형/rectangle-with-a-number.py
n = int(input())

# Please write your code here.
def numrec(n):
    cnt=1
    for _ in range(n):
        for _ in range(n):
            if cnt != 10:
                print(f"{cnt}",end=' ')
                cnt+=1
                
            else:
                cnt=1
                print(f"{cnt}", end=' ')
                cnt+=1
        print()
                
    

numrec(n)

            

File: 250312/숫자로 이루어진 사각형/rectangle-with-a-number.cpp
#include <iostream>

using namespace std;

int n;

int main() {
    cin >> n;
    int cnt = 1;
    // Please write your code here.
    for(int i=0; i<n; i++){
        for(int j=0; j<n; j++){
            if(cnt != 10){
                cout << cnt << " ";
                cnt+=1;
            }
            else{
                cnt = 1;
                cout << cnt << " ";
                cnt += 1;
            }
        }
        cout << endl;
    }

    return 0;
}

